p2p/node/localnodestore.go

```diff
@@ -141,15 +102,17 @@ func getLocalNodes(path string) ([]string, error) {
 // ReadFirstNodeData reads node data from the data folder.
 // Reads a random node from the data folder if more than one node data file is persisted.
 // To load a specific node on startup - users need to pass the node id using a cli arg.
-func ReadFirstNodeData(path string) (LocalNode, error) {
+//
+// TODO(josebalius): verify that "reads a random node" is actually happening
```

Comment:
The doc here might need updating, I don't see how this function is reading a "random" node from the data folder. We seem to fetch them all and read the first one. Does it need updating?

Reply:
yes you are correct, it is documenting older behavior..

Reply:
Thanks, I'll update the comment and remove the TODO.